Packs/Active_Directory_Query/TestPlaybooks/active_directory_search_user_with_parentheses_test.yml

```diff
@@ -0,0 +1,220 @@
+id: active directory search user with parentheses test
```

Comment:
* make sure the context is valid after running the command
* create a user with parentheses in the middle of the name and search for it